src/Auth0/Login/LoginServiceProvider.php

```diff
@@ -64,6 +64,12 @@ public function register()
         \Event::listen('Illuminate\Auth\Events\Logout', function () {
             \App::make('auth0')->logout();
         });
+        
+        $this->mergeConfigFrom(config_path('auth0.php'), 'auth0');
+        
+        if (file_exists(config_path('auth0-laravel.php')) {
+            $this->mergeConfigFrom(config_path('auth0-laravel.php'), 'auth0');
```

Comment:
I don't know where this file name is coming from, was previously `auth0-laravel.php`. Also: setting name.

Reply:
So here I'm merging the old config into the new key for backwards compat

Reply:
`auth0-laravel.php` is not the old settings file name ;)